eng/Tools.props

```diff
@@ -67,4 +61,8 @@
 
   </Target>
 
+  <Import Project="$(RepositoryEngineeringDir)docs.targets" Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
consider putting these targets in a subfolder or using some naming convention so it makes it clear they only participate in up-front restore.